README.md

```diff
@@ -83,5 +82,7 @@ $ docker compose up
 
 Access an interactive API documentation and a collection index explorer in a web browser:
 
-* API: http://localhost:8000/docs
-* UI: http://localhost:8001/
+- API: http://localhost:8000/docs
```

Comment:
Doesn't look like we need to change anything on this file, no?

Reply:
Not really